server/rpc.go

```diff
@@ -77,7 +77,7 @@ func (orch *orchestrator) GetJob(jid int64) (*lpTypes.Job, error) {
 		ta, err := orch.node.Eth.AssignedTranscoder(job)
 		if err != nil {
 			glog.Errorf("Could not get assigned transcoder for job %v, err: %s", jid, err.Error())
-			// continue here without a valid transcoder address
+			return nil, err
```

Comment:
I'm a bit unsure about this one; it seems like it would lead to unintended side effects.

For example, we're OK if there's a spurious error in acquiring the transcoder address during `verifyToken` , since the token has already been signed by the orchestrator (so we assume the job is valid). The job is only needed on the transcoder to ensure it hasn't expired. Returning an error here would prevent the transcoder from processing a segment, when we still want it to process the segment.

Note that we already already check if the assigned transcoder does not match the current transcoder in `verifyTranscoderReq`. If that check passes on the orchestrator, the orchestrator signs a token which the transcoder trusts.